packages/proto-loader/bin/proto-loader-gen-types.ts

```diff
@@ -617,7 +645,7 @@ function generateLoadedDefinitionTypes(formatter: TextFormatter, namespace: Prot
 
 function generateRootFile(formatter: TextFormatter, root: Protobuf.Root, options: GeneratorOptions) {
   formatter.writeLine(`import type * as grpc from '${options.grpcLib}';`);
-  formatter.writeLine("import type { ServiceDefinition, EnumTypeDefinition, MessageTypeDefinition } from '@grpc/proto-loader';");
```

Comment:
Note that ServiceDefinition in the old version of line appears to _always_ be unused in the generated code.

Reply:
Right, I think it became unused in #1745.